### PR TITLE
PAYARA-1331 Admin console integration for eventbus notifier

### PR DESCRIPTION
**Pull request by Pandrex247**
Requires https://github.com/payara/Payara/pull/1320 to be completed first, otherwise a merge conflict will likely happen.

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test please

File: appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/notification/email/GetEmailNotifierConfiguration.java [hunk 1]
```diff
@@ -77,4 +78,9 @@ protected String listConfiguration(EmailNotifierConfiguration configuration) {
         columnFormatter.addRow(values);
         return columnFormatter.toString();
     }
-}
\ No newline at end of file
+
+    @Override
+    protected Map<String, Object> getNotifierConfiguration(EmailNotifierConfiguration nc) {
```
Comment (smillidge) [inline appserver/payara-appserver-modules/notification-email-core/src/main/java/fish/payara/notification/email/GetEmailNotifierConfiguration.java]: You've got an UnSupportedOperationException have you missed adding this method implementation?

File: nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/log/GetLogNotifierConfiguration.java [hunk 2]
```diff
@@ -75,4 +76,9 @@ protected String listConfiguration(LogNotifierConfiguration configuration) {
         columnFormatter.addRow(values);
         return columnFormatter.toString();
     }
+
+    @Override
```
Comment (smillidge) [inline nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/log/GetLogNotifierConfiguration.java]: An unimplemented method body

**Comment by Pandrex247** [comment]
The unimplemented methods will be implemented in the coming PRs

**Pull request merged**